.travis.yml

```diff
@@ -15,16 +30,27 @@ before_install:
   - nvm alias default node
   - ./setup.sh
 
-install: true
+install:
+   # Install all the dependecies required to
+   # generate and validate code examples based
+   # on API specification in `raml2markdown/src`.
+   - ./scripts/code-examples/install-deps.sh
 
 script:
+  # Generate code examples from RAML files.
+  - ./scripts/code-examples/generate.sh
+  # Make sure that HOST is up and responds.
+  - ./scripts/code-examples/api-check.sh || travis_terminate 1
```

Comment:
@svenlaater do we still need this? moreover, this script just checks if `product` API is accessible, so it's not a best indicator that all is working correctly. If something configured in a bad way, we'll see it in the logs of validator tool I guess

Reply:
removed.